### ISSUE TITLE
JSON serialization of Explanation

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#75]
I think it makes sense to add something like ``asdict`` method to Explanation that will return a JSON-serializable object (it will just call ``attr.asdict(self)``).
And also add test that check that it is indeed json-serializable (right now it can have some numpy ints that are not seriazable).

**Comment by kmike** [comment]
Yeah, that'd be a great feature!
It also has a full-fledged numpy array in TransitionFeatureWeights.

**Comment by lopuhin** [comment]
Fixed in #84
